ui/src/components/Map.tsx

```diff
@@ -48,7 +65,7 @@ function Map(props: PropsWithChildren<IProps>) {
     >
       <TileLayer
         attribution='&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'
```

Comment:
Should we make the `attribution` configurable as well? Using different tiles might meen a different attribution needs to be displayed.